utils/__tests__/validate_packs.test.js

```diff
@@ -182,6 +182,16 @@ describe('test validateFile', () => {
 
       expect(errors.length).toBe(1);
     });
+
+    test('ensure error output is correct for null description', () => {
+      const dashboardTestFile = getTestFile('dashboard');
+      dashboardTestFile.contents[0].description = null;
+
+      const { errors } = validateFile(dashboardTestFile);
+
+      expect(errors.length).toBe(1);
+      expect(errors[0].message).toBe("'/description' must be string");
```

Comment:
would it be overkill to clearly state this is the dashboard description just to make it a little easier to resolve?

Reply:
the error does point to the specific file that failed, so that context should indicate its a dashboard.

if we wanted the word dashboard for this one field, that might be awkward, but is doable.

Reply:
That's cool. thanks for explaining!